bin/gen_PKGBUILD.py

```diff
@@ -0,0 +1,44 @@
+import tomli
+
+with open("pyproject.toml", 'rb') as f:
+    pyproject = tomli.load(f)
+
+description = pyproject["project"]["description"]
+version = open('anesthetic/_version.py','r').readlines()[0].split("=")[1].strip().strip("'")
+url = pyproject["project"]["urls"]["Homepage"]
+pyproject["project"]["dependencies"]
+rel=2
+
+
+PKGBUILD = """# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/wjhandley)
```

Comment:
```suggestion
PKGBUILD = """# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/{secrets.AUR})
```

Reply:
Although I would concede that the added complexity might not be worth it in this particular case, because when you hand it over to someone else, the other person should take care of this.